### PR TITLE
improve and update the readme file and correct the grammatical mistake

### PR DESCRIPTION
**Pull request by aaryan078**
**- What I did**
I found some grammatical mistake in the readme file so I correct them with the help of Grammarly

**- How I did it**
I correct them by my knowledge of grammar and Grammarly software

**- How to verify it**

**- Description for the changelog**
 grammatical mistake

pexels-photo-3690511

### CODE REVIEW

**Comment by aaryan078** [comment]
please someone review it

File: README.md [hunk 1]
```diff
@@ -6,33 +6,37 @@ The Moby Project
...
 
 ## Principles
 
-Moby is an open project guided by strong principles, aiming to be modular, flexible and without too strong an opinion on user experience.
+Moby is an open project guided by strong principles, aiming to be modular, flexible, and without too strong an opinion on user experience.
 It is open to the community to help set its direction.
 
 - Modular: the project includes lots of components that have well-defined functions and APIs that work together.
-- Batteries included but swappable: Moby includes enough components to build fully featured container system, but its modular architecture ensures that most of the components can be swapped by different implementations.
+
```
Comment (thaJeztah) [inline README.md:17]: This doesn't look correct; there should be no empty line between bullets

File: README.md [hunk 2]
```diff
@@ -6,33 +6,37 @@ The Moby Project
...
+- Batteries included but swappable: Moby includes enough components to build a fully-featured container system, but its modular architecture ensures that most of the components can be swapped by different implementations.
+
```
Comment (thaJeztah) [inline README.md:19]: Same here (and the ones below)

File: README.md [hunk 3]
```diff
@@ -6,33 +6,37 @@ The Moby Project
...
-However, other projects are also encouraged to use Moby as an upstream, and to reuse the components in diverse ways, and all these uses will be treated in the same way. External maintainers and contributors are welcomed.
+However, other projects are also encouraged to use Moby as an upstream and to reuse the components in diverse ways, and all these uses will be treated in the same way. External maintainers and contributors are welcomed.
 
 The Moby project is not intended as a location for support or feature requests for Docker products, but as a place for contributors to work on open source code, fix bugs, and make the code more useful.
-The releases are supported by the maintainers, community and users, on a best efforts basis only, and are not intended for customers who want enterprise or commercial support; Docker EE is the appropriate product for these use cases.
+The releases are supported by the maintainers, community, and users, on a best efforts basis only, and are not intended for customers who want an enterprise or commercial support; Docker EE is the appropriate product for these use cases.
```
Comment (thaJeztah) [inline README.md:39]: The `an` doesn't look correct here; "users who want enterprise or commercial support" is correct.

That said, `Docker EE` no longer exists, so we should probably remove those bits from this file.

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks for contributing; I had a quick look at the changes, but the commit is currently missing a DCO sign-off, and there's various changes that don't look correct.

This file needs a cleanup overall, so with only the two punctuation changes remaining in this PR, I think the changes are too marginal for that, so I'll go ahead and close this.

**Pull request closed without merging**